himportscan/src/HImportScan/ImportScanner.hs

```diff
@@ -60,8 +61,15 @@ instance Aeson.ToJSON ModuleImport where
 -- | Retrieves the names of modules imported in the given
 -- source file. Runs the GHC lexer only as far as necessary to retrieve
 -- all of the import declarations.
-scanImportsFromFile :: FilePath -> IO ScannedImports
-scanImportsFromFile filePath = scanImports filePath <$> Text.readFile filePath
+-- If a file is missing, we return 'Nothing'.
+-- TODO: It would be better to give more information on the missing file,
```

Comment:
I would do this here, but I would need to alter the JSON format with which go and hs communicate, so I'd prefer this to be done in a separate PR.